### ISSUE TITLE
Peer Review HW-09 for Ziqiangt

### ISSUE THREAD

**Issue by janehuang1647** [STAT545-UBC-students/hw09-ziqiangt#1]
Peer Review HW-09 for Ziqiangt
----------------------------------------------------
Topic                       | Excellent | Satisfactory | Needs Work |
----------------------------|-----|-----|-----|
**Coding style**            |  |:heavy_check_mark:  |  |
**Coding strategy**         |  |:heavy_check_mark:  |  |
**Presentation: graphs**    |:heavy_check_mark:  | | |
**Presentation: tables**    |:heavy_check_mark:    |  | |
**Achievement, creativity** | | :heavy_check_mark:  |  |
**Ease of access**          |:heavy_check_mark:  |  |  |

Remarks:
----------------------------------------------------
* README file has clear descriptions of the files generated from the makefile. 
* I like the diagram which clearly showed the dependencies of the file using the makefile 👍 .
* Graphs in the final output html file is adequately shown, but the link for the html file in the table is missing. You might forget to link this to the table 😢 . 
*  One small suggestion, to calculate the frequency of the letter, you can first capitalize or turn all letters into lower case, this would give a better result on the letter frequency (so far you have shown two separate column for capital and lower case letters)
*  The peer prview link is very convinient! 
*  Overall, you have done excellently for this assignment! 👍

**Comment by ziqiangt** [comment]
Yes, it is much better to use "str_to_lower()".  I agree with you.
Thanks a lot.
